addon/index.js

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
Having a single export module at a`addon/index.js` is not unheard of, even `ember-computed-decorators` does it: https://github.com/rwjblue/ember-computed-decorators/blob/master/addon/index.js